h/browser/chrome/lib/sidebar-injector.js

```diff
@@ -3,6 +3,32 @@
 var blocklist = require('../../../static/scripts/blocklist');
 var errors = require('./errors');
 var settings = require('./settings');
+var util = require('./util');
+
+var CONTENT_TYPE_HTML = 'HTML';
+var CONTENT_TYPE_PDF = 'PDF';
+
+// a function which is executed as a content script
+// to determine the type of content being displayed in a tab
```

Comment:
Maybe it's just me, but I prefer to distinguish "docstrings" (i.e. the docs for a function or something) from normal inline code comments, by writing them as a jsdoc or ngdoc comment (or in Python as a docstring). Python is probably where this comes from actually: you wouldn't put a normal code comment before a Python function saying what it does, because that's what a docstring is for. Does the same apply to jsdoc comments? (I know we're never going to want to use jsdoc to extract it, but it's just for clarity.)

The other great thing about Python docstrings is that PEP257 actually describes how they should be written to be clear and consistent. Basically sentences that start with "Return " are good. With this function for example, you would write "Return 'PDF' if the document is a PDF, 'HTML' otherwise. This function is executed as a content script in a tab's document". I think that's much clearer. With the comment as currently written, I actually have to read the code to figure out what the function does / returns.